docker-compose.test.yaml

```
@@ -92,6 +92,38 @@ services:
+      ZOOKEEPER_TICK_TIME: 2000
+    ports:
+      - "2182:2182"
+
+  kafka-test:
+    image: confluentinc/cp-kafka:latest
+    depends_on:
+      - zookeeper-test
+    profiles:
+      - infra
+    healthcheck:
+      test: [ "CMD", "kafka-topics", "--bootstrap-server", "localhost:9092", "--list" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)